src/app/app.component.spec.ts

```diff
@@ -49,5 +52,27 @@ describe('AppComponent', () => {
         })
       })
     })
+
+    // describe('setDefaultForm', () => {
+    //     it('Should reset the form if touched property is not true', async () => {
+    //       // data.resetForm(hello)
+    //       console.log(data)
+    //       fixture.detectChanges();
+    //       component.setDefaultForm(data);
+    //       expect(data).toBe({wind: 10, rain: 20, vis: 20000});
+    //     })
+    //   })
+
+    //outdated test placeholder til button updated
+    // describe('btnClick', () => {
+    //   it('Should Add objects to an object', async () => {
+    //     component.weatherInputs = {wind: 5};
+    //     component.precipitationInput = 50;
+    //     component.visibilityInput = 15;
+    //
+    //     component.btnClick();
+    //     expect(component.weatherInputs).toBe({wind: 5, rain: 50, vis: 15});
+    //   })
+    // })
```

Comment:
Are those the tests that you are stuck on?

Reply:
Yes, well setDefaultForm im stuck on. 
btnClick currently does nothing, I forgot to remove the test contents